prboom2/src/d_main.c

```diff
@@ -322,6 +322,9 @@ void D_Display (fixed_t frac)
     oldgamestate = -1;            // force background redraw
   }
 
+  if (V_IsOpenGLMode() && !exclusive_fullscreen && !dsda_SkipMode())
```

Comment:
Is skip mode in particular relevant here? Does `!nodrawers` satisfy the condition?

Reply:
Most likely. I didn't look too far into it, I was just making sure I wouldn't step on SkipMode's toes by performing glClear on framebuffers that weren't going to have any work done to them.